static/app/views/dashboardsV2/widget/index.tsx

```diff
@@ -13,7 +13,7 @@ type Props = React.ComponentProps<typeof WidgetBuilder>;
 function WidgetBuilderContainer({organization, ...props}: Props) {
   return (
     <Feature
-      features={['metrics', 'dashboards-edit']}
+      features={['metrics-dashboards-ui', 'dashboards-edit']}
```

Comment:
```suggestion
      features={['metrics', 'metrics-dashboards-ui', 'dashboards-edit']}
```
Otherwise all requests fail with 404